Portfolio: setup and mobile version skeleton

Bondok6 opened the pull request:
Built header section that contains:
   - Navigation
          - title
          - menu 
   - Hero section
           - main title
           - description
           - social icons

Bondok6 committed:
add linters files

- .hintrc - stylelintrc.json

Bondok6 committed:
Add files

- index.html - main.css - .gitignore

Bondok6 committed:
add images folder and MIT.md file

Bondok6 committed:
built header section

- navigation - hero-section

Bondok6 committed:
updated Readme.md file

- add screenshot

andersonlebon commented on index.html:
```diff
@@ -0,0 +1,58 @@
+          <img src="./images/menu.svg" alt="menu-icon" class="nav__menu">
+        </div>
+      </nav>
+
```
- [x] According to requirements of this milestone and HTML semantic tags this `<header>` should contain only the logo and the navigation `<nav>`
- [x] Logo design and placement, as well as the presence of a text link to the homepage affect the success of navigation to the homepage. Kindly make this  Logo a link, use the`<a href=""></a>` instead of the `<h2></h2>`
This is the format you can use for the `<header>` 
````
<header>
      <a href=" "> Welcome </a>
      <nav >
          <img src="humbager" alt="">
      </nav>
</header>

````

andersonlebon commented on index.html:
```diff
@@ -0,0 +1,58 @@
+            </a>
+          </li>
+        </ul>
+      </div>
```
- [x] By applying HTML semantic tags. I think it will be nice if we put this part of code in a `<section></section>` instead of a `<div>` tag. This will improve the skeleton of the HTML page.
- [x] Good job on implementing the requirement by using `<ul>` congrats :clap: however I think you forgot some of the links. Kindly add the `Angelist` and `Medium`link. check here down.

| your design | the template|
| --------------- | ---------------- |
|  ![Screenshot from 2021-11-25 04-55-59](https://user-images.githubusercontent.com/65068771/143372095-356c8e9a-a3fc-4c62-a263-2e9340e3eab7.png) | ![Screenshot from 2021-11-25 04-56-22](https://user-images.githubusercontent.com/65068771/143372117-5bd18d7a-ee8c-46c7-a4c0-1078f0f61371.png) |
- [x] I know you have worked hard implementing this style however I think it would be nice if you can follow the same `paddings` and `margins` and `font-sizes` that are used in the template. Check here :point_down: 

| your design | the template|
| --------------- | ---------------- |
| ![Screenshot from 2021-11-25 05-16-12](https://user-images.githubusercontent.com/65068771/143373658-6ba5e355-ed28-43ba-b90b-c89637571b83.png) | ![Screenshot from 2021-11-25 04-50-17](https://user-images.githubusercontent.com/65068771/143372900-e137a597-f2c1-4b38-9d43-fecae5729f2a.png) |

- [x] At the bottom of your design I think you forgot to put the down button. Kindly let's add it as the template is designed. check :arrow_down: 
![Screenshot from 2021-11-25 04-59-57](https://user-images.githubusercontent.com/65068771/143374025-c7f5a956-8d6a-47ae-8250-e94e88773e78.png)

andersonlebon reviewed (changes requested):
Hi @Bondok6  

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://user-images.githubusercontent.com/65068771/141811112-252360b9-757f-4279-bb3b-b7a2a4675ad3.gif)
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Bondok6 committed:
update header section

- used an anchor tag instead of h2 tag - header contain only logo and the navigation

Bondok6 committed:
add to links

- added angellist and medium icons - put section instead of div

Bondok6 committed:
added down-button

Bondok6 committed:
Add prefixer CSS and update MIT.md file

Bondok6 committed:
update padding

Bondok6 committed:
update the order of the links

Cathella reviewed (approved):
## 🟢 STATUS: APPROVED

### Hi @Bondok6 👋🏻 , 

You have done well implementing all the requested changes 👏🏻!
Your project looks perfect!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
